sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_search_sessions_v1/schema.yaml

```diff
@@ -0,0 +1,110 @@
+fields:
+- mode: NULLABLE
+  name: submission_date
+  type: DATE
+- mode: NULLABLE
+  name: glean_client_id
+  type: STRING
+- mode: NULLABLE
+  name: event_timestamp
+  type: INTEGER
+- mode: NULLABLE
+  name: normalized_channel
+  type: STRING
+- mode: NULLABLE
+  name: normalized_country_code
+  type: STRING
+- mode: NULLABLE
+  name: event_name
+  type: STRING
+- mode: NULLABLE
+  name: search_session_type
+  type: STRING
+- mode: NULLABLE
+  name: engaged_result_type
+  type: BOOLEAN
+- mode: NULLABLE
+  name: annoyance_signal_type
+  type: STRING
+- mode: NULLABLE
+  name: first_result_type
+  type: STRING
+- mode: NULLABLE
+  name: number_of_chars_typed
+  type: INTEGER
```

Comment:
Despite this field being an integer, when I order this field in Bigquery, it is sorting it as a string.